ip_country/src/ip_country.rs

```diff
@@ -75,22 +73,27 @@ fn generate_country_data(
     output: &mut dyn io::Write,
 ) -> Result<(), io::Error> {
     let bit_queue_len = bit_queue.len();
-    write!(output, "\n")?;
```

Comment:
I cannot comment on this directly but line 64 contains a TODO with a suggestion tightly coupled with the card being reviewed which normally is not an acceptable state. Because it's not clear either to me what Dan talks about in this comment I advise you to consult together the future of the referred idea.